src/common/__tests__/cluster-store.test.ts

```diff
@@ -446,3 +446,13 @@ describe("pre 3.6.0-beta.1 config with an existing cluster", () => {
     expect(icon.startsWith("data:;base64,")).toBe(true);
   });
 });
+
+describe("getClusterIdFromHost", () => {
+  it("should return undefined for non cluster frame hosts", () => {
+    expect(getClusterIdFromHost("localhost:45345")).toBeUndefined();
+  });
+
+  it("should return undefined for non cluster frame hosts", () => {
+    expect(getClusterIdFromHost("fe540901-0bd6-4f6c-b472-bce1559d7c4a.localhost:59110")).toBe("fe540901-0bd6-4f6c-b472-bce1559d7c4a");
```

Comment:
We need test also for `xyz.fe540901-0bd6-4f6c-b472-bce1559d7c4a.localhost:59110` because watch api uses random subdomain as a workaround for http1.1 limitations

Reply:
Good test case, thanks